target/linux/ramips/dts/mt7621_oraybox_p5.dts

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
```

Comment:
Add header `#include <dt-bindings/leds/common.h>` here. And please do a build test.